bin/wsdump.py

```diff
@@ -116,6 +116,8 @@ def recv_ws():
         while True:
             opcode, data = recv()
             msg = None
+            if six.PY3 and type(data) is bytes:
```

Comment:
I think the following would be better.

```python
if six.PY3 and opcode == websocket.ABNF.OPCODE_TEXT and isinstance(data, bytes):
    data = str(data, "utf-8")
```

1. we need to convert data only if the data is TEXT type.
2. TEXT data is utf-8 encoded. "ENCODING" is terminal encoding.